src/utils/messageEmbed.ts

```diff
@@ -8,7 +8,8 @@ import ollama, { ChatResponse } from 'ollama'
  * @param msgHist message history between user and model
  */
 export async function embedMessage(
-    message: Message, 
+    message: Message,
+    ollama: Ollama,
```

Comment:
Why does embedMessage need a reference to the ollama object?? I don't see where it is used below.

Reply:
Refer to line 35, the `ollama.chat(...)` it is the same as before, however it does not change as only the importing changed because we want to use a custom client as the port changes from `127.0.0.1` to a custom one from the container.